Library/Homebrew/formula_auditor.rb

```diff
@@ -150,14 +150,6 @@ def audit_formula_name
       problem "Formula name conflicts with existing core formula."
     end
 
-    PROVIDED_BY_MACOS_DEPENDS_ON_ALLOWLIST = %w[
-      apr
-      apr-util
-      libressl
-      openblas
-      openssl@1.1
-    ].freeze
```

Comment:
Code this get a `CODEOWNERS` added in Homebrew/brew to request review from me and the TSC? We need to ensure that we only add entries here that are definitely shipped on macOS (as we've had maintainers add to this list incorrectly in the past).

Reply:
No problem! I'll handle it in https://github.com/Homebrew/homebrew-core/pull/65712

Reply:
Done